snuba/query/physical.py

```diff
@@ -0,0 +1,5 @@
+from snuba.query.query import Query
+
+# This defines the type for the physical query that operates on storages.
+# Obviously this is supposed to become more complex than just a type alias.
```

Comment:
What are the differences that we'd expect to see here between the physical and logical query class?

Reply:
- clickhouse specific stuff like pre-where would not be in the logical query
- tags as mapping columns `tags[asdasd]` are very likely not going to be in the physical query.

There may be more.